content/blog/contributor-highlight-francesco-canovai/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Contributor Spotlight: Francesco Canovai"
+date: 2025-05-07T14:51:43+02:00
+draft: false
+image:
+    url: 
+    attribution:
+author: fdrees
+tags:
+ - testing
+ - github
+ - kubernetes
+ - postgresql
+ - open-source
+summary: "In a mini-series on this blog we highlight the work of the community.
+  Today we meet Francesco Canovai, Senior Software Development Engineer at EDB, 
+  and CloudNativePG maintainer." 
+---
+
+# Contributor Spotlight: 
+
+Building and maintaining and open source project takes a village. In a
+mini-series on this blog we would like to highlight the work of our
+maintainers, component owners, and members of the larger community.
+
+Today it's my great pleasure to introduce [Francesco Canovai](https://github.com/fcanovai/), 
+my colleage at EDB. Francesco is a Senior Software Development Engineer, 
+living in Prato, Italy. No point asking when he first found out about 
+CloudNativePG, he was in the room when/where the project started! The 
+first commit with his name on it is the initial suite of e2e tests.
+
+One dream he has for the project is to rework the tests to use a configuration 
```

Comment:
"a configuration" ... requires a bit more explanation not to sound magic.
I think he's talking about dagger.io or perhaps there is a more generic
name for that type of framework.

Reply:
@fcanovai can you help out here and elaborate?

Reply:
I mean, I'd like to get rid of all the environment variables that we use, and have a clear configuration file to define which tests to run and how to run them. Way easier than going rummaging through the GitHub workflow, the Makefile and the shell scripts.

It would make things easier for new developers, it could allow to testing custom environments, it would make easier for me to understand where a few parameters come from when the tests are run.

And of course will require an overhaul of the tests  :)

Reply:
OK, we've had some debate but no actual proposal of text.
Here's one, wdyt @fcanovai?:
```suggestion
One dream he has for the project is to rework the tests so they are driven by
a single unified configuration file, instead of the Frankenstein approach in current
use entailing GH actions, Makefiles ... 
```